tests/main/local-install-w-metadata/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: Checks for local install with metadata from assertions
+# XXX we would need to bother with curl there atm
+systems: [-ubuntu-core-16-64]
+execute: |
+    echo "Get the snap and its assertions"
+    snap download test-snapd-tools
+    # XXX snap download should do this as well
+    curl -H "Accept: application/x.ubuntu.assertion" -o snap-decl.assertion https://assertions.ubuntu.com/v1/assertions/snap-declaration/16/eFe8BTR5L5V9F7yHeMAPxkEr2NdUXMtw
+    # XXX have a 'snap snap-digest' command?
```

Comment:
Probably. Let's wait until we figure how we want to name this (hash? digest? sha3-384?).